src/extensions/windows/Microsoft.DotNet.UpgradeAssistant.Extensions.Windows/Templates/WinAppSDKTemplates/UWPToWinAppSDKUpgradeHelpers.cs

```diff
@@ -16,4 +16,16 @@ interface IDataTransferManagerInterop
             [System.Runtime.InteropServices.In] ref Guid riid);
         void ShowShareUIForWindow(IntPtr appWindow);
     }
+
+    static class DataTransferManagerHelper
+    {
+        public static Windows.ApplicationModel.DataTransfer.DataTransferManager MarshalledDataTransferManagerFromWindow(IntPtr appWindow)
```

Comment:
Consider GetForWindow or GetDataTransferManagerForWindow.

Also consider this implementation: https://github.com/microsoft/WindowsAppSDK/issues/1335#issuecomment-985023124